src/Compilers/CSharp/Portable/Generated/Syntax.xml.Internal.Generated.cs

```diff
@@ -39995,6 +39995,7 @@ public InterpolatedStringExpressionSyntax InterpolatedStringExpression(SyntaxTok
       {
         case SyntaxKind.InterpolatedStringStartToken:
         case SyntaxKind.InterpolatedVerbatimStringStartToken:
+        case SyntaxKind.VerbatimInterpolatedStringStartToken:
```

Comment:
i slightly worry about this because code may not expect to have to check for both of these.  I think i'd prefer a single kind, where the text is something you check.  Note: we have precedence for this ```@""``` and ```""``` strings are the same SyntaxKind.  You just check if its verbatim by checking the first char.